### ISSUE TITLE
Route tags for crag listing

### ISSUE THREAD

**Issue by birgander2** [theCrag/website#2304]
I always liked a lot the route tags in Rockfax topo publications, describing some basic route's characteristics. [Here is an example](http://www.rockfax.com/databases/results_buttress.html?id=1095) with tags for "crimpy", "pumpy", "powerful", "technical", "scary"; missing there is "reachy". It would be great have something similar, ideally shown in the overview crag listing.

Also, it would be nice to additionally have tags for "photo" and "video" there, which are shown when a route has a photo or video linked. This requires of course an additional field in the route's mask (or a scan for linked videos in the description field).

**Comment by scd** [comment]
@birgander2 would your tag examples be in addition to our standard tags. How would you categorise this group of route characteristics? (ie our tags need a group header)

The advantage of doing it this way is we get language translations for these standard tags for free.

Photos are already linkable and you can embed a photo. This is for images on our site.

Of course there are external resource images and videos. Videos are embeddable, but it would be better to have it in the data model itself. This is something we plan to implement.

**Comment by birgander2** [comment]
Yes, in addition, but (additionally) shown in the crag listing, not only in the route's description, just similar to the Rockfax example. In fact, they give a nice overview of a crag's style, help selecting routes to climb, etc. I hope it's more clear now what I mean? Possible group header could be "style", "demands"!? (your english knowledge is certainly better...)

Same for a photo and video tags. Currently, one has to click through all the routes to check if there is something "hidden". It would be very convenient to have these two tags in the crag's list as an indication.

**Comment by Drazhar** [comment]
This should also, and even more important, be done for areas.

It would be great to directly see the "sun" tags (all day, shady, ...), the climbing style (slab, vertical, ...) and the kid friendlyness in the area overview.

**Comment by Mdemaillard** [comment]
We do have a few of these tag already, but not all. There are pretty much hidden by the sheer number of rocktypes tags in the same field.
![image](https://user-images.githubusercontent.com/60388146/76863790-be7d6d80-6868-11ea-9d9c-35f5ff6b40f9.png)

For the display, that opens a few questions: 
- which tag do you display in the listing for a route ? (all would be too long)
 do you display inherited tag from the area in a route ? (probably not)
- Which tag for an area ? 
do you roll up in an area ? (probably not. if yes, under which criteria ? (1 route has the tag ? a majority of routes have the tag ?)

Different users will be more interested in different tags, possibly set different rules between boulder/trad/sport? (Or even user defined in profile (all tag listed, tick some that you want to see?)

Objectively on Desktop for routes there is space for a few tags in list view.
Also on Desktop for areas, if we replace the current text for Sport/trad/boulder by the tag used in the route, that would create some space.
![image](https://user-images.githubusercontent.com/60388146/76864574-fc2ec600-6869-11ea-8f27-e72733a9ef61.png)

Now for Phone that's another beast of its own.

**Comment by Mdemaillard** [comment]
merging example picture made by Drazhar on #2358

**Comment by georg-d** [comment]
> We do have a few of these tag already, but not all. There are pretty much hidden by the sheer number of rocktypes tags in the same field.

IMHO we may move the rocktype tags from "Route character" into their very own, dedicated dropdown. This will make the remaining "Route character" dropdown much more overseeable and the rock types do not tell nearly as much as the other tags (e.g. overhung or vegetated or suggested powerful or technical) about the charakter of a route, e.g. limestone can range from one solid mass to a Swiss cheese with 1000 holes, from antiskid to totally slippery & polished, from very loose to really stable.
